### ISSUE TITLE
Settings kubernetes in yaml.schemas gives error

### ISSUE THREAD

**Issue by JPinkney** [redhat-developer/yaml-language-server#202]
As of 0.6.0 it looks like setting kubernetes in yaml.schemas leads to an error [here](https://github.com/redhat-developer/yaml-language-server/blob/master/src/server.ts#L263)

**Comment by CNLHC** [comment]
same problem here, the traceback is 

```
TypeError: jsonSchemaService_1.FilePatternAssociation is not a constructor
    at isKubernetes (/home/cn/.vscode-server/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:199:21)
    at validateTextDocument (/home/cn/.vscode-server/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:228:62)
    at Timeout._onTimeout (/home/cn/.vscode-server/extensions/redhat.vscode-yaml-0.6.0/node_modules/yaml-language-server/out/server/src/server.js:217:9)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7
```

**Comment by JPinkney** [comment]
@CNLHC Should be fixed in the newest version (0.6.1)
